typedoc.json

```diff
@@ -1,11 +1,9 @@
 {
     "out": "docs/",
-    "inputFiles": "src/",
+    "entryPoints": ["src/"],
     "exclude": "src/codec/**/*.ts",
-    "excludeNotExported": true,
     "excludeExternals": true,
-    "stripInternal": true,
-    "ignoreCompilerErrors": true,
+    "excludeInternal": true,
     "excludePrivate": true,
     "excludeProtected": true
```

Comment:
Refer to https://github.com/TypeStrong/typedoc/releases/tag/v0.20.0 for reasoning behind this change